extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -271,19 +280,88 @@ void ConsumeWindowsEventLog::onTrigger(const std::shared_ptr<core::ProcessContex
     return;
   }
 
-  const auto flowFileCount = processQueue(session);
+  struct TimeDiff {
+    auto operator()() const {
+      return std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now() - time_).count();
+    }
+    const decltype(std::chrono::steady_clock::now()) time_ = std::chrono::steady_clock::now();
```

Comment:
Use `auto` instead of `decltype(std::chrono::steady_clock::now())` for simplicity. They are equivalent in this case, since we work with values, not references.

Reply:
`auto` for class members allowed for `static` only.